CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Contributing to Mello
+
+:tada: Thanks for contributing to the project! :tada:
+
+The following is a guide for contributing to Mello. These are guidelines, not rules, so use your best judgement! Suggestions for changes are welcome in the form of a pull request.
+
+## Table Of Contents
+### [Code of Conduct](#code-of-conduct)
+
+### [Contributing to the Project](#contributing-to-the-project)
+ * [Reporting Bugs](#reporting-bugs)
+ * [Suggesting Enhancements](#suggesting-enhancements)
+ * [Pull Requests](#pull-requests)
+
+### [Project Resources](#project-resources)
+ * [Adding Background Images](#adding-background-images)
+ * [Adding SVG Images](#adding-svg-images)
+
+### [Styleguides](#styleguides)
+ * [Naming Conventions](#naming-conventions)
+ * [BEM Naming for CSS](#bem-naming-for-css)
+ * [File Structure](#file-structure)
+ * [Git Commit Messages](#git-commit-messages)
+
+### [Acknowledgements](#acknowledgements)
+
+## Code of Conduct
+This project and everyone participating in it is governed by the [Mello Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code.
+
+## Contributing to The Project
+### Reporting Bugs
+Bugs are tracked through GitHub issues, to report a bug, create an issue and provide the following information:
+* A Description of the Bug.
+* Steps to reproduce the bug.
+* Any necessary additional information.
+
+Bugs should be appropriately labelled as such.
+### Suggesting Enhancements
+Enhancements are tracked through GitHub issues, to suggest an enhancement, create an issue and provide the following information:
+* Provide a clear and informative title to identify the suggestion.
+* Provide a detailed description of the suggested enhancement.
+* Explain why this enhancment would be useful to most Mello users.
+
+Enhancements should be appropriately labelled as such.
+### Pull Requests
+
+## Project Resources
+### Adding SVG Images
+### Adding Background Images
+
+## Styleguides
+### Naming Conventions
+Names of project files should:
+* BeWrittenInPascalCase.
```

Comment:
Names of components should be in PascalCase and end in .jsx. 
Names of function files should be in camelCase and end in .js. 
Names of scss files should also be in camelCase.

Reply:
Appreciate the extra detail, will add this